primitives/linux-port/column.cpp

```diff
@@ -1123,6 +1139,7 @@ void scalarFiltering_(
   T curValue = 0;
   primitives::RIDType rid = 0;
   bool isEmpty = false;
+bool ourCase = filterCount == 2 && ((uint64_t)filterValues[0]) == 0xfeffffffffffffffULL && ((uint64_t)filterValues[1]) == 0x31235247464dULL;
```

Comment:
You need to clean ourCase from the whole hierarchy now :)

Reply:
Done.